src/cases/schemas/responses/find-case-response.schema.js

```diff
@@ -1,15 +1,14 @@
 import Joi from "joi";
-import { CasePayload, CaseStage } from "../case.schema.js";
+import { CaseStage } from "../case.schema.js";
 import { UrlSafeId } from "../url-safe-id.schema.js";
 
 export const findCaseResponseSchema = Joi.object({
-  _id: Joi.object().required(),
+  _id: Joi.string().hex().length(24).required(),
```

Comment:
Previously returned an instance of `ObjectId`. Now `ObjectId` is contained in repos and the `Case`s and `Workflow`s deal with strings.